packages/react-scripts/template/src/App.scss

```diff
@@ -19,4 +19,8 @@
   &__main {
     padding: $bpk-spacing-base 0;
   }
+
+  &__text {
```

Comment:
`. App__text` is declared once but used twice (in the react template). Ideally there should be a one to one mapping of a class to a react element. E.g. imagine we wanted to make the heading underlined:
```scss
  &__text {
    margin-bottom: $bpk-spacing-sm;
    text-decoration: underline;
  }
```
We've just inadvertently added underline to the paragraph. I reckon we should be setting the right example in the template.